packages/react-core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -3,12 +3,14 @@ import { css, getModifier } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/EmptyState/empty-state';
 
 export enum EmptyStateVariant {
+  extraLarge = 'extraLarge',
```

Comment:
nitpick: most components use the short-hand names, sm, lg, xl, etc. Example: EmptyStateIcon
we should try to be uniform i think. Unfortunately we already have large and small spelled out :) What if there is a xxl variant next? Maybe `xl = 'xl'` would be better?

Reply:
Would this resolve your concern @dlabrecq? I'll open a bug for this inconsistency; maybe it can get fixed in an upcoming breaking changes release.

Reply:
Is the suggestion to use 'xl'?

Yes, I'd rather introduce the name we want now (i.e., use 'xl') and change the remaining props during the breaking change release.

Reply:
@rebeccaalpert can you open an issue to update the remaining prop names during breaking change.  You can put the `breaking change` label on it.

@dlabrecq @jschuler what would `full` be in shorthand?  I don't think we have that one in other components.

Reply:
@tlabaj - I opened this on Wednesday: https://github.com/patternfly/patternfly-react/issues/3873 (just added the label, thanks).

Reply:
Thanks Rebecca!

Reply:
Looking at the 'full' variant and it appears be an empty string. It could be 'xxl', or possibly omitted?